### ISSUE TITLE
Live share visual studio plus screenity extension results in white screen

### ISSUE THREAD

**Issue by gregkonush** [alyssaxuu/screenity#13]
thanks for putting work into this extension, when I try to record desktop video I don't have any of the drawing tools available for me, only within a browser, maybe it would be impossible to have a tool outside of the browser window. The second issue is when I try to record a live share session from the visual studio website, the whole website becomes blank, maybe the way DOM gets manipulated is conflicting the way vscode live is working

![image](https://user-images.githubusercontent.com/12027037/99919764-43603700-2cd4-11eb-962c-bf4bd206c898.png)

**Comment by alyssaxuu** [comment]
Unfortunately yes, the controls can only be injected in Chrome, so they're not available in other apps. As per Visual Studio going blank, can you go into Inspect Element and the console tab to see if there's any errors being thrown?

**Comment by gregkonush** [comment]
I guess one of the libraries is getting 404?

![image](https://user-images.githubusercontent.com/12027037/99920109-36dcde00-2cd6-11eb-97f3-e3e639fa6f6a.png)

**Comment by alyssaxuu** [comment]
Nah that's a "normal" warning, unrelated to the issue though. Very odd, maybe there's some sort of "protection" added to the site to prevent anything from being injected, maybe due to the iframes loading it's causing it to crash. Does the website go back to normal after you stop the recording?

**Comment by gregkonush** [comment]
after I stop recording, I have to refresh the page to get back to the normal state

**Comment by alyssaxuu** [comment]
Hmm yeah it just seems like there's something that gets triggered that crashes it, and not the extension's CSS or anything messing around. Not sure how to fix it really, I don't have access to the page myself to investigate further ^^

**Comment by gregkonush** [comment]
you can try to reproduce it yourself with visual studio code editor if you use it, all you have to do is to start live share and use the browser as the second participant

**Comment by alyssaxuu** [comment]
I looked into it and I don't think there's anything I can do in my end unfortunately. There seems to be something in place to prevent elements being added to the DOM. I tried even removing the injected script manually afterwards but it just stayed broken and no matter what I did the screen was just white. I'll have to close this as there doesn't seem to be any workaround whatsoever, Visual Studio themselves would have to allow that.
